app/forms/update_user_profile_form.rb

```diff
@@ -16,16 +16,18 @@ def persisted?
 
   def initialize(user)
     @user = user
-    self.mobile = @user.mobile
+    self.mobile = @user.formatted_mobile
```

Comment:
Why this? The mobile is already formatted below in `set_attributes`. If I revert this change, the tests still pass.